keanu-project/src/main/java/io/improbable/keanu/tensor/dbl/Nd4jDoubleTensor.java

```diff
@@ -25,11 +30,8 @@
 
 public class Nd4jDoubleTensor implements DoubleTensor {
 
-    static {
-        System.setProperty("dtype", "double");
-    }
-
     public static Nd4jDoubleTensor scalar(double scalarValue) {
+        Nd4j.setDataType(DataBuffer.Type.DOUBLE);
```

Comment:
Pull this `DataBuffer.Type.DOUBLE` out into a public static?

Reply:
Yup good call.